ORBIT/phases/design/semi_submersible_design.py

```diff
@@ -61,53 +65,56 @@ def run(self):
 
     @property
     def stiffened_column_mass(self):
-        """
-        Calculates the mass of the stiffened column for a single
-        semi-submersible in tonnes. From original OffshoreBOS model.
+        """Calculates the mass of the stiffened column for a single
+        semi-submersible in tonnes [1].
         """
 
         rating = self.config["turbine"]["turbine_rating"]
         mass = -0.9581 * rating**2 + 40.89 * rating + 802.09
+        mass = -0.9581 * rating**2 + 40.89 * rating + 802.09
```

Comment:
Is it me or is this a duplicate line?

Reply:
Yup. That is strange... Might have copied something on accident... Fixed.